server/schema.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE users (
+  id                    SERIAL PRIMARY KEY,
```

Comment:
For SQL syntax we generally want to use what is known as screaming-snake-case for our column names. So for example if we had a column `firstName` in screaming snake case it would be `FIRST_NAME`